### PR TITLE
2022 06 technical review updates

### PR DESCRIPTION
**Pull request by khamui**
because some of the dep versions recommended by renovate do not work in conjunction with others and some update PRs are missing, i find it easier to update all deps in one single commit. Once this is okay, we can close or rerun renovate to remove the open PRs.

### CODE REVIEW

File: patches/jest-expo+45.0.1.patch [hunk 1]
```diff
@@ -0,0 +1,12 @@
+diff --git a/node_modules/jest-expo/src/preset/assetFileTransformer.js b/node_modules/jest-expo/src/preset/assetFileTransformer.js
+index c4b0952..dcda667 100644
+--- a/node_modules/jest-expo/src/preset/assetFileTransformer.js
++++ b/node_modules/jest-expo/src/preset/assetFileTransformer.js
+@@ -1,6 +1,6 @@
+ const createCacheKeyFunction = require('@jest/create-cache-key-function').default;
+ 
+ module.exports = {
+-  process: (_, filename) => `module.exports = 1;`,
++  process: (_, filename) => ({code: `module.exports = 1;`}),
```
Comment (khamui) [inline patches/jest-expo+45.0.1.patch:10]: i apply this patch with `patch-package`. Not sure how to install a commit/version of a subpackage though..

**Comment by phillimorland** [comment]
It installs however complains about the following, do we want to deal with this in another PR? 
ansi-regex  3.0.0 || 4.0.0 - 4.1.0
`Severity: high
 Inefficient Regular Expression Complexity in chalk/ansi-regex - https://github.com/advisories/GHSA-93q8-gq69-wqmw
 Inefficient Regular Expression Complexity in chalk/ansi-regex - https://github.com/advisories/GHSA-93q8-gq69-wqmw
fix available via `npm audit fix`
node_modules/@testing-library/jest-native/node_modules/ansi-regex
node_modules/inquirer/node_modules/ansi-regex
node_modules/inquirer/node_modules/strip-ansi/node_modules/ansi-regex

minimist  <1.2.6
Severity: critical
Prototype Pollution in minimist - https://github.com/advisories/GHSA-xvch-5gv4-984h
fix available via `npm audit fix`
node_modules/minimist

plist  <3.0.5
Severity: critical
Prototype pollution in Plist - https://github.com/advisories/GHSA-4cpg-3vgw-4877
fix available via `npm audit fix`
node_modules/plist

simple-plist  <1.3.1
Severity: critical
Prototype Pollution in simple-plist - https://github.com/advisories/GHSA-gff7-g5r8-mg8m
fix available via `npm audit fix`
node_modules/simple-plist`

**Comment by khamui** [comment]
@phillimorland i tried to replicate in my local and checked the build for dev, but cannot see this warning. Maybe add an issue for now?

**Review by goapunk — APPROVED** [review]
very nice, thanks for dealing with the update hell! I pushed a commit to make the web version work again

**Pull request merged**